localgov_publications.module

```diff
@@ -61,6 +71,18 @@ function localgov_publications_theme_suggestions_book_navigation(array $variable
   return $suggestions;
 }
 
+/**
+ * Implements hook_block_access().
+ */
+function localgov_publications_block_access(Block $block, $operation, AccountInterface $account) {
+  if ($block->getPluginId() == 'localgov_page_header_block' && $operation == 'view') {
+    $node = \Drupal::routeMatch()->getParameter('node');
+    if (!empty($node) && $node instanceof NodeInterface && $node->getType() == 'localgov_publication_page') {
```

Comment:
Is the check on empty() necessary? I'd have thought checking $node is an instance of NodeInterface here is enough.